server/backend/migration/20230826000000.go

```diff
@@ -0,0 +1,66 @@
+package migration
+
+import (
+	"database/sql"
+
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/guregu/null"
+	"gorm.io/gorm"
+)
+
+type Feedback struct {
```

Comment:
Why define this type here again? Just for consistency of the migration I'm guessing?

Reply:
Correct.

Reply:
Yes, if I migrate while referncing the model, this would change may change the meaning of the migration in the future